### PR TITLE
i18n: Translations update from MAA Weblate

### PR DESCRIPTION
**Pull request by AlisaAkiron**
Translations update from [MAA Weblate](https://weblate.maa-org.net) for [MAA Assistant Arknights/Glossary](https://weblate.maa-org.net/projects/maa/glossary/).

It also includes following components:

* [MAA Assistant Arknights/WPF GUI](https://weblate.maa-org.net/projects/maa/wpf-gui/)

Current translation status:

Weblate translation status

## Summary by CodeRabbit

- **文档更新**
	- 优化了中英文本地化文件的文本内容，提升了清晰度和特异性。
	- 格式更新，使 `<ResourceDictionary>` 标签从多行格式合并为单行格式，提升了可读性。此更改不影响应用程序的功能或逻辑。
	- 对日文和中文（简体和繁体）的本地化字符串进行了调整，进一步改善了翻译的准确性和一致性。

### CODE REVIEW

**Comment by Constrat** [comment]
The never ending war between webplate formatting and our formatting. Is there really no solution?

File: src/MaaWpfGui/Res/Localizations/en-us.xaml [hunk 1]
```diff
@@ -857,7 +854,7 @@ With save file: farm points through crafting (advanced).
   3. If the quantity that can be manufactured is exactly a multiple of 99, MAA can get confused (stuck). You can use up a little before saving. This might be repaired later.
     </system:String>
     <system:String x:Key="ReclamationBePatient">Chill and relax, please be patient</system:String>
-    <system:String x:Key="Reclamation2ExEnable">Earn points by manufacturing</system:String>
+    <system:String x:Key="Reclamation2ExEnable">farm points through crafting</system:String>
```
[the remark this answers is not in the document]
Comment (Constrat) [reply src/MaaWpfGui/Res/Localizations/en-us.xaml:857]: Why do people ignore initial capital, this makes me go crazy.

**Comment by Constrat** [comment]
@zkywalker230061 In the context of translating you can't be just translating what you see. It's a bit more complex where you need to know where that string is going to be.
Removing the `\n` from the Roguelike output doesn't make sense in EN, as the strings are very long compared to CN and putting them in line is impossible, pushing the numbers over disrupting the correct display of the output.

Please ensure that the first letter is always capitalized. There must be no room for error. Webplate should serve as assistance, not as a burden requiring constant checks for introduced errors.

**Review by Constrat — APPROVED** [review]

File: src/MaaWpfGui/Res/Localizations/en-us.xaml [hunk 2]
```diff
@@ -857,7 +854,7 @@ With save file: farm points through crafting (advanced).
   3. If the quantity that can be manufactured is exactly a multiple of 99, MAA can get confused (stuck). You can use up a little before saving. This might be repaired later.
     </system:String>
     <system:String x:Key="ReclamationBePatient">Chill and relax, please be patient</system:String>
-    <system:String x:Key="Reclamation2ExEnable">Earn points by manufacturing</system:String>
+    <system:String x:Key="Reclamation2ExEnable">Farm points through crafting</system:String>
```
[the remark this answers is not in the document]
Comment (Constrat) [reply src/MaaWpfGui/Res/Localizations/en-us.xaml:857]: @coderabbitai there's no API key here. It's just a xaml for localization purposes

[the remark this answers is not in the document]
Comment (Constrat) [reply src/MaaWpfGui/Res/Localizations/en-us.xaml:857]: Holy shit this is incredbible!

**Comment by zkywalker230061** [comment]
Got it, thanks.



------------------ Original ------------------
From: Constrat ***@***.***&gt;
Date: Sat,Aug 10,2024 5:10 AM
To: MaaAssistantArknights/MaaAssistantArknights ***@***.***&gt;
Cc: CY-Zhu ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [MaaAssistantArknights/MaaAssistantArknights] i18n: Translationsupdate from MAA Weblate (PR #10178)





 
@zkywalker230061 In the context of translating you can't be just translating what you see. It's a bit more complex where you need to know where that string is going to be.
 Removing the \n from the Roguelike output doesn't make sense in EN, as the strings are very long compared to CN and putting them in line is impossible, pushing the numbers over disrupting the correct display of the output.
 
Please ensure that the first letter is always capitalized. There must be no room for error. Webplate should serve as assistance, not as a burden requiring constant checks for introduced errors.

**Comment by Constrat** [comment]
And it was changed again. And we even have conflicts lol

**Comment by Constrat** [comment]
DIY solution: Consolidated all changes from this PR into a single direct commit to resolve the conflict. It's not the ideal approach, but it certainly functions. I'm beginning to consider restricting direct access to the webplate and the ability to "translate," as errors or issues occur 9 out of 10 times.

**Comment by zkywalker230061** [comment]
> DIY solution: Consolidated all changes from this PR into a single direct commit to resolve the conflict. It's not the ideal approach, but it certainly functions. I'm beginning to consider restricting direct access to the webplate and the ability to "translate," as errors or issues occur 9 out of 10 times.

The conflict was raised by another commit #10182, where the CN string was changed. Also because of this weblate is locked and cannot be merged again :(

**Comment by Constrat** [comment]
_Let's leave it at that_
@zkywalker230061 if you want to make changes. Just create a PR. That way we can work on conflicts manually and to be fair. You can see the xaml in itself.

**Comment by zkywalker230061** [comment]
> _Let's leave it like that_ @zkywalker230061 if you want to make changes. Just create a PR. That way we can work on conflicts manually and to be fair. You can see the xaml in itself.

Indeed, that’s what it used to be.

**Comment by ABA2396** [comment]
> The never ending war between webplate formatting and our formatting. Is there really no solution?

ee360c3ddf41651b32a1829297a15a8dff2c0538 fixed

**Pull request closed without merging**